jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sqlserver/jbpm-7.59-to-7.60.sql

```diff
@@ -3,3 +3,4 @@ alter table BAMTaskSummary drop column end_date;
 alter table TaskEvent drop column end_date;
 alter table NodeInstanceLog drop column end_date;
 alter table VariableInstanceLog drop column end_date;
+alter table NodeInstanceLog add column observation varchar(255);
```

Comment:
`column` is an invalid identifier in mssql - please remove
```suggestion
alter table NodeInstanceLog add observation varchar(255);
```

Reply:
ok